01-import_and_maxfilter.py

```diff
@@ -91,8 +91,7 @@ def run_maxwell_filter(subject, session=None):
 
         if run_idx == 0:  # XXX does this work when no runs are specified?
             # Prepare the pipeline directory in /derivatives
-            deriv_path = op.join(config.bids_root, 'derivatives',
-                                 config.PIPELINE_NAME)
+            deriv_path = config.deriv_root
```

Comment:
I would do:

deriv_path = config.deriv_path

or 

deriv_root = config.deriv_root

pick your poison

Reply:
I made it that way because in all scripts except the first one we do:
```python
    deriv_path = op.join(config.deriv_root, subject_path)
```
I could rename `deriv_path` to `deriv_root` in the first script?

Reply:
Also just to mention, once https://github.com/mne-tools/mne-bids/pull/410 has been merged, we can simplify this code further, as we won't have to manually construct so many paths anymore :)

Reply:
Addressed in ac19b7a

Reply:
I've unified things now in the `01` script to work like in the others: 95c7d09

Reply:
Haha wow what a mess :) Should be good now!